### PR TITLE
Missing endpoint for marking a single notification as read

### PR DESCRIPTION
**Pull request by wkigenyi**
## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/notification/service/NotificationReadPlatformServiceImpl.java [hunk 1]
```diff
@@ -107,6 +107,13 @@ public void updateNotificationReadStatus() {
         this.jdbcTemplate.update(sql, appUserId);
     }
 
+    @Override
+    public void updateSingleNotificationReadStatus(Long id) {
+        String sql = "UPDATE notification_mapper SET is_read = true WHERE is_read = false AND id = ?";
```
Comment (galovics) [inline fineract-provider/src/main/java/org/apache/fineract/notification/service/NotificationReadPlatformServiceImpl.java:112]: I know the existing method uses native query but please don't introduce it in one more method.
Please rely on JPA for updating the entity. Thanks.

**Review by galovics — CHANGES_REQUESTED** [review]
Please use JPA to update the entity and introduce a new integration test-case to verify this API functionality. Thanks.

**Comment by galovics** [comment]
@wkigenyi is this going to be fixed or not? Thanks.

**Comment by wkigenyi** [comment]
Most probably not in 1.8, will fix in 1.9
Thanks @galovics

**Comment by galovics** [comment]
@wkigenyi I'll close the PR then. Please raise another one when the comment is fixed. Let me know if you need guidance.

**Pull request closed without merging**